StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/SpinnerView.swift

```diff
@@ -13,28 +13,21 @@ final class SpinnerView: UIView {
 
     init() {
         super.init(frame: .zero)
-        let activityIndicator = ActivityIndicator(size: .large)
-        activityIndicator.color = .iconActionPrimary
-        activityIndicator.backgroundColor = .customBackgroundColor
-        activityIndicator.startAnimating()
-        addSubview(activityIndicator)
 
-        // `ActivityIndicator` is hard-coded to have specific sizes, so here we scale it to our needs
-        let largeIconDiameter: CGFloat = 37
-        let desiredIconDiameter: CGFloat = 44
-        let transform = CGAffineTransform(
-            scaleX: desiredIconDiameter / largeIconDiameter,
-            y: desiredIconDiameter / largeIconDiameter
-        )
-        activityIndicator.transform = transform
-        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
+        let imageView = UIImageView(image: Image.spinner.makeImage())
```

Comment:
this is a very simplistic implementation but I believe its good enough for first iteration